__tests__/app.test.js

```diff
@@ -483,3 +483,131 @@ describe("/api/users", () => {
     });
   });
 });
+
+describe("GET /api", () => {
```

Comment:
I probably wouldn't bother testing this much on this bit. 
This test is very brittle and will break everytime you complete and endpoint and add another thing to the file. 
I'd just check it works and you get an object back, rather than all the content